tests/test_api.py

```diff
@@ -103,8 +103,7 @@ def test_raw_default_query_delta_with_end(self, grep):
         resp = self.client.get("/raw?end=1564503781")
         self.assertEqual(resp.status_code, 200)
         kws = grep.call_args[1]
-        # Verify the default query delta was not applied
-        self.assertNotEqual((kws["end"] - kws["start"]).total_seconds(), 180.0)
+        self.assertEqual((kws["end"] - kws["start"]).total_seconds(), 180.0)
```

Comment:
The test was wrong?

Reply:
previously, we were using the hard-coded constant of 600 seconds in utils here -- now it uses the default_query_delta, so we acutually need to check that it is using that now

Reply:
i tried and failed to use current_app, running into issues wiht the flask application context, so i also tried to fix that and failed.

added it as an extra commit if you have suggestions, but TBH, would like to know if this is worth it to just avoid passing an extra parameter to the util function.

Reply:
Ah I see, thanks